bin/improver-percentile

```diff
@@ -63,16 +68,23 @@ def main():
                         "coordinates to create percentiles, but is redundant "
                         "when converting probabilities to percentiles and may "
                         "be omitted.")
-    parser.add_argument("--percentiles", metavar="PERCENTILES",
-                        nargs="+", default=None, type=float,
-                        help="Optional definition of percentiles at which"
-                        " to calculate data, otherwise default values are"
-                        " used, e.g. --percentiles 0 33.3 66.6 100 ; "
-                        " defaults = [0, 5, 10, 20, 25, 30, 40, 50, 60, 70,"
-                        " 75, 80, 90, 95, 100]")
+    group = parser.add_mutually_exclusive_group(required=False)
+    group.add_argument("--percentiles", metavar="PERCENTILES",
+                       nargs="+", default=None, type=float,
+                       help="Optional definition of percentiles at which to "
+                       "calculate data, e.g. --percentiles 0 33.3 66.6 100")
+    group.add_argument('--no_of_percentiles', default=None, type=int,
```

Comment:
it is typical to 'hyphenante' the option names, so I would use `--no-of-percentiles`

Reply:
Done.